tests/cli/main.cpp

```diff
@@ -1084,3 +1086,99 @@ BOOST_AUTO_TEST_CASE( cli_create_htlc )
    }
    app1->shutdown();
 }
+
+static string encapsulate( const graphene::wallet::signed_message& msg )
+{
+   fc::stringstream encapsulated;
+   encapsulated << "-----BEGIN BITSHARES SIGNED MESSAGE-----\n"
+                << msg.message << '\n'
+                << "-----BEGIN META-----\n"
+                << "account=" << msg.meta.account << '\n'
+                << "memokey=" << std::string( msg.meta.memo_key ) << '\n'
+                << "block=" << msg.meta.block << '\n'
+                << "timestamp=" << msg.meta.time << '\n'
+                << "-----BEGIN SIGNATURE-----\n"
+                << fc::to_hex( (const char*)msg.signature->data, msg.signature->size() ) << '\n'
+                << "-----END BITSHARES SIGNED MESSAGE-----";
+   return encapsulated.str();
```

Comment:
Note: this code looks like duplicate of https://github.com/bitshares/bitshares-core/pull/1878/files#diff-eba6dcca79f28def3dea11f284600592R2747, however, having this in test cases (here) exactly does the testing job.

Reply:
Agree. Can't think of a better solution right now.